pages/glossary/default-page-language.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Default Page Language
+key: default-page-language
+unambiguous: true
+objective: true
+input_aspects:
+  - DOM tree
+  - Accessibility tree
+  - CSS Styling
+  - Language
+---
+
+The default page language is the _most common language_ used in a [web page][]. If there are no _words_ in the page, the default language is undefined.
+
+The _most common language_ is determined by counting the number of _words_ in the [web page][] that are part of any of the languages in the [language subtag registry][]. The same word can be part of multiple languages. If the number of words is the same, the default language is undefined.
+
+The _words_ used in the following texts should be counted for the _most common language_:
+
+- **page title**: The value of the [document title][] of the [document][] in the [top-level browsing context][].
+- **default language content**: The value of any [text nodes][] and the [accessible name][] of any [elements][] for which all of the following is true:
```

Comment:
I think there should be examples where the accessible name plays a role in determining the language.
```html
<span id="explanation" aria-hidden=true>this word has no English equivalent</span>
<span aria-labelledby="explanation">gelukkig</span>
```
or something similar.

Reply:
Not too keen on an aria-labelledby one, but an image makes sense. I'll work on something.